www/klei00/semester_project_1/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="cs">
+<!-- Web dostupný z: https://eso.vse.cz/~klei00/ -->
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Bitcon</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Dodavatel technologií a spotřebního materiálu">
+    <meta name="keywords" content="bitcon,signmaking,spotřební materiál,tiskárny,plotry,inkousty">
+    <meta name="author" content="Iveta Kleníková">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen, print" href="styles/main.css">
+    <link rel="icon" href="images/favicon.png" type="image/png">
+</head>
+
+<body>
+    <header>
+
+        <div class="logo">
+            <span>Bitcon</span>
+        </div>
+        <nav>
+            <ul>
+                <li><a href="news.html">Aktuality</a></li>
+                <li><a href="about_us.html">O&nbsp;nás</a></li>
+                <li><a href="price_list.html">Ceník</a></li>
+                <li><a href="contacts.html">Kontakty</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <div class="intro">
+            <h1>Váš dodavatel technologií a spotřebních materiálů</h1>
+        </div>
+        <div class="content">
+            <div class="main_offer">
+                <ul>
+                    <li class="product_group">
+                        <h2>Tiskové stroje</h2>
+                        <p>V oblasti velkoformátových tiskáren nabízíme technologie prémiové značky Roland. Tato firma
+                            jako
+                            první na světě uvedla na trh jedinečnou patentovou technologii kombinovaného plotru.</p>
+                        <form action="price_list.html#printers">
+                            <button type="submit">PŘEHLED PRODUKTŮ</button>
```

Comment:
takhle pomoci form action to vidim poprve v zivote, neco tak jednoducheho vyzaduje jednoduche reseni, tj pomoci `<a href="...">` , pokud jde o stylu, `<a>` muze taky vypadat jako `<button>` pomoci CSS

pomoci form je to spatne, protoze ti vrazi otaznik do URL v nove otevrene strance